models/friends.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose');
+const Joi = require('joi');
+
+const friendSchema = new mongoose.Schema({
+    User_id: { type: Number, required: true },
+    Requested_by:{type: Number, required: true},
+    Online: { type: Boolean, required: true },
+    FriendStatus: { type: String, required: true }
+})
+
+const FriendStatus = mongoose.model('FriendStatus', friendSchema);
```

Comment:
Lars, Change mongoose.model('FriendStatus', friendSchema) TO mongoose.model('Friends', friendSchema);